### DISCUSSION TITLE
How to use multiple components with the same event handler

### DISCUSSION THREAD

**Discussion by santroshg** [JohannesKlauss/react-hotkeys-hook#1052] [Q&A]
How to use multiple components with the same event handler?

I have a grid (parent component) made of 10 cell and each cell (child component ) added hook and callback in cell (child component ), so it is trigging 10 times. 

Is there a way to run callback only once inside child component.

**Comment by JohannesKlauss** [comment]
You can use the ref that is returned from the callback to bound the execution when an element is focused.

See this doc page: https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/scoping-hotkeys
